ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -71,7 +75,21 @@ Item {
 
                 readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
                 readonly property bool isHovered: memberItem.sensor.containsMouse
-                readonly property bool canBeBanned: !memberItem.itsMe && (model.memberRole !== Constants.memberRole.owner && model.memberRole !== Constants.memberRole.admin)
+                readonly property bool canBeBanned: {
+                    if (memberItem.itsMe) {
+                        return false
+                    }
+                    switch ( model.memberRole) {
+                        // Owner can't be banned
+                        case Constants.memberRole.owner: return false
+                        // TokenMaster can only be banned by owner
+                        case Constants.memberRole.tokenMaster: return root.isOwner
+                        // Admin can only be banned by owner and tokenMaster
+                        case Constants.memberRole.admin: return root.isOwner || root.isTokenMaster
+                        // Everyone can be banned by everyone
+                        default: return true
```

Comment:
Is it really the case?

Reply:
Oh, well the comment is not good, It should say "Everyone else can be banned by privileged users", since this panel is only visible by Admins, etc.